packages/picasso-codemod/src/v17.0.0/typography-sizes/typography-sizes.ts

```diff
@@ -10,8 +13,14 @@ const getNewSize = (oldSize: string): string => {
       return oldSize
   }
 }
-
-let affectedComponents = ['Typography', 'TypographyOverflow', 'Amount']
+// exclude components with `variant` prop other than body
+const shouldBeExcluded = (attributes: JSXAttribute[]) =>
```

Comment:
I'd rename, so we can avoid the comment with the explanation.

```suggestion
const hasVariantBodyAttribute = (attributes: JSXAttribute[]) =>
```

Reply:
I updated the function a little bit so I could use this naming which I like - it's `hasVariantHeadingAttribute`